app/src/components/wallet/PagePortfolio.vue

```diff
@@ -1,23 +1,31 @@
 <template>
-  <TmPage :sign-in-required="true" :managed="true" :dark-background="true">
-    <template slot="managed-body">
+  <TmPage :sign-in-required="true" :dark-background="true">
```

Comment:
does it make sense to have `:sign-in-required="true"` and `<template slot="signInRequired">` ?

i think it does but there might be a smarter / simpler way.

Reply:
why the extra slot? `CardSignInRequired` should replace the managed content

Reply:
which slot is extra?

Reply:
@faboweb ping